Cargo.toml

```diff
@@ -10,9 +10,10 @@ license = "GPL-3.0-or-later"
 
 [dependencies]
 anyhow = "1.0.79"
-gpiod = "0.2.3"
+gpiocdev = "0.7.1"
 mio = { version = "0.8.10", features = ["net", "os-poll", "os-ext"] }
 nix = { version = "0.27.1", features = ["signal", "time"] }
+chrono = "0.4"
```

Comment:
`chrono` is a good library, but it's overpowered for just grabbing timestamps. Simple timestamping can be done with the tools in [`std::time`](https://doc.rust-lang.org/std/time/).